checkstyle/suppressions.xml

```diff
@@ -116,6 +120,12 @@
               files="JsonConverter.java"/>
     <suppress checks="NPathComplexity"
               files="DistributedHerder.java"/>
+    <suppress checks="NPathComplexity"
```

Comment:
`ConnectHeader` needed a suppression when it had all of the conversion methods, but since those are gone I can remove that suppression. `ConnectHeaders` and `Values` still are complex enough that they need a suppression.